airflow/models/dag.py

```diff
@@ -804,6 +804,12 @@ def get_is_paused(self, session=None) -> Optional[None]:
         qry = session.query(DagModel).filter(DagModel.dag_id == self.dag_id)
         return qry.value(DagModel.is_paused)
 
+    @provide_session
+    def get_is_active(self, session=None) -> Optional[None]:
+        """Returns a boolean indicating whether this DAG is active"""
+        qry = session.query(DagModel).filter(DagModel.dag_id == self.dag_id)
+        return qry.value(DagModel.is_active)
+
```

Comment:
Please don't "split" get_is_paused and is_paused (move this up or down one function.)